src/components/Tickets.js

```diff
@@ -8,18 +8,20 @@ function Tickets() {
         <div className='tickets row text-center'>
           <div className="ticket ticket-1 col-sm box-shadow">
             <h4 className="ticket-header">EARLY-BIRD PASS</h4>
-            <p className="ticket-price">K 50</p>
-            <p className="ticket-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
+            <p className="ticket-price">K 150</p>
+            <p className="ticket-content">Available from 29th July to 19th August 2019.</p>
+            <a href="https://qkt.io/I3SHAd"><p className="btn btn-info btn-sm">Click here to Purchase.</p></a>
           </div>
-          <div className="ticket ticket-2 col-sm box-shadow">
+          {/*<div className="ticket ticket-2 col-sm box-shadow">
             <h4 className="ticket-header">STUDENT PASS</h4>
             <p className="ticket-price">K 80</p>
             <p className="ticket-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id mauris ex.</p>
-          </div>
-          <div className="ticket ticket-3 col-sm box-shadow">
+  </div>*/}
+          <div className="ticket ticket-2 col-sm box-shadow">
             <h4 className="ticket-header">REGULAR PASS</h4>
-            <p className="ticket-price">K 100</p>
-            <p className="ticket-content">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis id mauris ex.</p>
+            <p className="ticket-price">K 250</p>
+            <p className="ticket-content">Available from 20th August to 20th September 2019.</p>
```

Comment:
for some reason the text here is going beyond the div margins

Reply:
I'll check it out. I thought I fixed this when I made the background image a bit more responsive so that it scales with the protruding text

Reply:
Whats the hold up on the PR now? @OlivierJM

Reply:
It was the text that was overflowing but I replaced Available with Get that seems fine now.